lib/versions.js

```diff
@@ -60,3 +60,15 @@ function fetchLatestTag(repo) {
     },
   );
 }
+
+function getVersionFromToolVersionsFile() {
+  const fileName = ".tool-versions";
+  if (!fs.existsSync(fileName)) return;
+
+  const toolVersions = fs.readFileSync(fileName, "utf-8").split("\n");
+  const scarbVersionInfo = toolVersions.find((tool) => tool.match(/^scarb/));
```

Comment:
Nit: please add space, so we only match `scarb ` exactly. Can be just `.startsWith`.